### PR TITLE
Adding my blog to the list

### PR DESCRIPTION
**Pull request by manisuec**

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+   {
+    name: "Manish Prasad",
+    description:
+      "Founding Team Member | 👨🏻‍💻 Developer | ✍️ Blogger",
+    url: "https://techinsights.manisuec.com",
+    twitter: "@_manish25",
+    tags: [
+      "React",
+      "Nodejs",
+      "Nodej.js",
```
Comment (naresh-io) [inline data.js:11]: Is that `Node.js`?

Comment (manisuec) [reply data.js:11]: yes, since both are treated as different tags

Comment (naresh-io) [reply data.js:11]: I mean there is a typo. It's not `Nodej.js`, should be `Node.js`

Comment (manisuec) [reply data.js]: updated

**Review by markodenic — APPROVED** [review]

**Pull request merged**